C/services/common/plugin_manager.cpp

```diff
@@ -393,7 +393,7 @@ char		buf[MAXPATHLEN];
   strncpy(buf, path.c_str(), sizeof(buf));
   if (buf[0] && access(buf, F_OK|R_OK) == 0)
   {
-	pluginHandle = new PythonPluginHandle(name.c_str(), buf);
+	pluginHandle = new SouthPythonPluginHandle(name.c_str(), buf);
```

Comment:
Is PluginManager::loadPlugin() called for loading north and storage plugins also?

Reply:
Currently PythonPluginHandle = South plugin interface.

So doing pluginHandle = new SouthPythonPluginHandle(name.c_str(), buf)

is just the same